about.html

```diff
@@ -0,0 +1,166 @@
+        </li>
+        <li class="top-header-item">
+          <a class="top-header-item-link" href="http://linkedin.com/in/raihan2bd"><img class="top-social-icon" src="./images/linkedin.png" alt="Linkedin"/></a>
+        </li>
+        <li class="top-header-item">
+          <a class="top-header-item-link" href="#">English</a>
+        </li>
+        <li class="top-header-item">
+          <a class="top-header-item-link" href="#">Mypage</a>
+        </li>
+        <li class="top-header-item">
+          <a class="top-header-item-link" href="#">Logout</a>
+        </li>
+      </ul>
+    </div>
+    <nav class="navbar">
+      <div class="menu-icon"><img src="./images/menu-icon.png" alt="menu" /></div>
+      <a class="brand" href="index.html"><img src="./images/logo.png" alt="Code With Raihan" /></a>
+      <!-- Navigation Item for mobile -->
+      <ul class="nav-item-group m-nav">
+        <span class="cross-icon"><img src="./images/cross-icon.png" alt="x" /></span>
+        <li class="nav-item">
+          <a href="index.html" class="nav-link">Home</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#program" class="nav-link">Program</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#tutor" class="nav-link">Tutors</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#sponser" class="nav-link">Sponser</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#hero" class="nav-link nav-box">Join Program</a>
+        </li>
+      </ul>
+      <!-- Navigation Items for Desktop -->
+      <ul class="nav-item-group d-nav">
+        <li class="nav-item">
+          <a href="index.html" class="nav-link">Home</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#program" class="nav-link">Program</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#tutor" class="nav-link">Tutors</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#sponser" class="nav-link">Sponser</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#hero" class="nav-link nav-box">Join Program</a>
+        </li>
+      </ul>
+  </header> <!-- Heder end here -->
+
+  <!-- Main container start Form here -->
+  <main class="container">
+
+    <!-- About Headline -->
+    <section id="about-headline" class="about-headline">
+      <div class="headline-block">
+      <h1 class="headline-title">Full Stack WebDevelopment<br> Bootcamp 2023</h1>
+      <p class="headline-details">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ducimus numquam sint laborum itaque quis aut recusandae dolores corporis nesciunt pariatur esse, veritatis, aperiam deserunt perferendis dignissimos facilis! Aut, saepe pariatur?</p>
+      <h5>Please Contack us if you need any furthur information about this Bootcamp</h5>
+      <address>example@example.com</address>
+    </div>
+    </section>
```

Comment:
**_kindly fix these design issues in your about.html header to match the design🙏  :_**

- [ ] Kindly Increase The Size of the main title to match the design 
- [ ] Kindly adjust the font weight for the email to be bolder so you match the design

your design : 
![image](https://user-images.githubusercontent.com/89282221/203383365-bedb0580-0022-4513-a6f3-7ed63d5e0a76.png)


required design : 
![image](https://user-images.githubusercontent.com/89282221/203383116-8193da57-0d1a-4c4a-a81a-64dde7774a38.png)